src/licensedcode/models.py

```diff
@@ -571,7 +571,7 @@ def validate(licenses, verbose=False, no_dupe_urls=False):
                 error(f'Unknown language: {lic.language}')
 
             if lic.is_unknown:
-                if not 'unknown' in lic.key:
+                if not 'unknown' in lic.key and not 'undetected' in lic.key:
```

Comment:
IMHO we should introduce soon after this a is_something flag for these licenses rather than to special case their name.

Reply:
We didn't add this after all, so resolving this.